protocol-definitions/custom/Custom.yaml

```diff
@@ -828,6 +828,10 @@ customTypes:
         type: Map_EndpointQualifier_Address
         nullable: false
         since: 2.0.1
+      - name: autoStepDownWhenLeader
```

Comment:
```suggestion
      - name: cpAutoStepDownWhenLeader
```

Reply:
agree that prefixing with `cp` makes it more understandable, especially for the future readers.